tests/android_only_country_selected.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+from appium import webdriver
+from tests import desired_capabilities
+
+
+class OnlyCountrySelectedTests(unittest.TestCase):
+
+    def setUp(self):
+        desired_caps = desired_capabilities.get_desired_capabilities()
+        self.driver = webdriver.Remote('http://localhost:4723/wd/hub', desired_caps)
+        self.driver.implicitly_wait(200)
+        self.driver.find_element_by_id("android:id/text1").click()
+        self.driver.implicitly_wait(200)
+        self.driver.find_element_by_android_uiautomator('text("greece")').click()
```

Comment:
Tests depend on data in API. Could you rewrite test to click e.g. first element in a list?